graphqlQueries/getUserById

```diff
@@ -0,0 +1,25 @@
+query getUserById($id: String!) {
```

Comment:
IMPORTANT: This query don't really work but we need it as example for the futur.

Need to discuss about the ID in mongodb and how we can use it or not.

Reply:
pour discussion :
@Field((type) => ID) (avec import ID from Type-graphql) 
readonly _id: ObjectId; (avec import ObjectId from mongodb et sans définit de @prop sur _id)

Reply:
Si c'est la solution pour pouvoir utiliser l'ID mongo dans notre appli, pourquoi pas !
A discuter donc. Et création d'un ticket dans la foulée si nécessaire.